source/gems/dry-validation/comparison-with-activerecord.html.md

```diff
@@ -5,49 +5,61 @@ name: types
+
+But, how would we go about converting our Active Record validation code into Dry-Validation?
+
+After reading this guide, you will know:
+
+- How to use dry-validation to replace built-in Active Record validation helpers.
+- How to use dry-validation to create your own custom validation methods.
+
 ## 1. Validation Overview
 
-When using ActiveRecord validation, you declare your validations in the model in the following format:
+When using ActiveRecord validation, validations are declared in the model in the following format:
 
 `validates :name, :email, presence: true`
 
-When using Dry Validation, you declare your validation in a separate schema class using predicates to build up rules.
+When using Dry-Validation, you declare your validation in a separate schema class using predicates to build up rules.
+
+A predicate is a simple stateless method which receives some input and returns either `true` or `false`.
+
 For example:
 
 ```ruby
-require 'dry-validation'
-
-class Schema < Dry::Validation::Schema
-  key(:email) { |email| email.filled? }
-  key(:name) { |name| name.filled? }
+Dry::Validation::Schema do
+  key(:email).required
+  key(:name).required
 end
 ```
-or the short-hand:
 
-```ruby
-require 'dry-validation'
+## 2. Validation Helpers
 
-class Schema < Dry::Validation::Schema
-  key(:email, &:filled?)
-  key(:name, &:filled?)
-end
-```
-You can declare multiple predicates per key (or attribute) for example
+### 2.1 acceptance
 
-Active Record: `validates :email, presence: true, format: { with: EMAIL_REGEX }`
+In ActiveRecord validations this helper is used to validate that a checkbox on the user interface was checked when a form was submitted. This is typically used when the user needs to agree to your application's terms of service, confirm reading some text, or any similar concept.
 
-Dry Validation: ` key(:email) { |email| email.filled? & email.format?(EMAIL_REGEX) }`
+In its simplest form:
 
-## 2. Validation Helpers
+<dl>
+  <dt>ActiveRecord Validation</dt>
+  <dd><pre>validates :attr, acceptance: true</pre></dd>
 
-### 2.1 acceptance
+  <dt>Dry Validation</dt>
+  <dd><pre>key(:attr) { bool? & true? }</pre></dd>
+</dl>
 
-| Active Record Validation                         | Dry Validation                         |
-|--------------------------------------------------|----------------------------------------|
-| `validates :attr, acceptance: true`              | `key(:attr){ |attr| attr.eql?('1') }`  |
-| `validates :attr, acceptance: { accept: 'yes' }` | `key(:attr){ |attr| attr.eql?('yes') }`|
+When using the `:accepts` option:
 
-Note: Active Record automatically creates the virtual acceptance attribute for you, so you will need to do this manually.
+<dl>
+  <dt>ActiveRecord Validation</dt>
+  <dd><pre>validates :attr, acceptance: { accept: 'yes' }</pre></dd>
```

Comment:
Could you try using "``` ruby ... ```" to get nice syntax highlighting inside the `dds`? It should look nice.